mesonbuild/compilers/compilers.py

```diff
@@ -441,34 +441,34 @@ class CompilerArgs(typing.MutableSequence[str]):
     # *always* de-dup these because they're special arguments to the linker
     always_dedup_args = tuple('-l' + lib for lib in unixy_compiler_internal_libs)
 
-    def __init__(self, compiler: typing.Union['Compiler', StaticLinker],
-                 iterable: typing.Optional[typing.Iterable[str]] = None):
+    def __init__(self, compiler: T.Union['Compiler', StaticLinker],
+                 iterable: T.Optional[T.Iterable[str]] = None):
         self.compiler = compiler
-        self.__container = list(iterable) if iterable is not None else []  # type: typing.List[str]
+        self.__container = list(iterable) if iterable is not None else []  # type: T.List[str]
 
-    @typing.overload
-    def __getitem__(self, index: int) -> str:
+    @T.overload
+    def __getitem__(self, index: int) -> str:  # noqa: F811
         pass
 
-    @typing.overload
-    def __getitem__(self, index: slice) -> typing.List[str]:
+    @T.overload
+    def __getitem__(self, index: slice) -> T.List[str]:  # noqa: F811
         pass
 
-    def __getitem__(self, index):
+    def __getitem__(self, index):  # noqa: F811
         return self.__container[index]
 
-    @typing.overload
-    def __setitem__(self, index: int, value: str) -> None:
+    @T.overload
+    def __setitem__(self, index: int, value: str) -> None:  # noqa: F811
         pass
 
-    @typing.overload
-    def __setitem__(self, index: slice, value: typing.List[str]) -> None:
+    @T.overload
```

Comment:
__[Flake8]__ redefinition of unused '__setitem__' from line 460 ([see on Sider](https://sider.review/gh/repos/19784232/pulls/6412#issue-5025067))

<details>

| Rule |
| - |
| F811 |

</details>
<sub>posted by Sider</sub>